src/bgen/Generator.cs

```diff
@@ -4863,6 +4863,9 @@ void GenerateProtocolTypes (Type type, string class_visibility, string TypeName,
 		var extensionMethods = optionalInstanceMethods.Concat (requiredInstanceMethods.Where (v => IsRequired (v, out var generateExtensionMethod) && generateExtensionMethod));
 		var extensionProperties = optionalInstanceProperties.Concat (requiredInstanceProperties.Where (v => IsRequired (v, out var generateExtensionMethod) && generateExtensionMethod));
 
+		// disable CS1573, which can happen when the original member in the api definition has xml comments and we copy that xml comment into the generated interface - because we may add parameters to method signatures, and the new parameters won't have an xml comment.
+		print ("#pragma warning disable CS1573"); // Parameter 'This' has no matching param tag in the XML comment for '...' (but other parameters do)
```

Comment:
Does it make sense to inject a comment as to why this is being disabled?

Reply:
I don't think most people look at the code we generate, so probably not worth it.